### ISSUE TITLE
[2.0.0 release] Bug report - Running decks doesn't work

### ISSUE THREAD

**Issue by TheNavigat** [TheTorProject/ooni-probe#592]
```
$ sudo ooniprobe -i deck.deck 
Unhandled Error
Traceback (most recent call last):
Failure: ooni.errors.OONIUsageError:

```

**Comment by hellais** [comment]
Can you give me the content of the deck you are trying to run it against?

**Comment by hellais** [comment]
Ok, I can confirm that I can reproduce this when running an older deck file. In ooniprobe 2.0.0 you should be using the newer version of the decks, however there was the intention of having backward compatibility with legacy decks so this is indeed a bug.

**Comment by hellais** [comment]
This should be fixed in the latest tip of the v2.0.0 branch.

Thanks so much for testing this and reporting this bug! 👍
